src/sentry/static/sentry/app/views/settings/account/accountNotificationFineTuning.tsx

```diff
@@ -313,7 +315,7 @@ export default class AccountNotificationFineTuning extends AsyncView<Props, Stat
               initialData={fineTuneData}
             >
               {isProject && hasProjects && (
-                <AccountNotificationsByProject projects={projects} field={field} />
+                <AccountNotificationsByProject projects={projects!} field={field} />
```

Comment:
I don't like to use `!` but in this case, it was necessary as the compiler doesn't satisfy  with `hasProjects`